Add telescope to docker-compose

vcfxb opened the pull request.

vcfxb committed:
Add telescope submodule

vcfxb committed:
Add telescope to caddy, docker

vcfxb committed:
remove Telescope submod

vcfxb committed:
Use telescope docker image in docker compose

rushsteve1 commented:
This needs a DNS rule set by @wdturner, and some more documentation on setting up the config (or a link to that in the Telescope repo)
Aside from those, looks good!

vcfxb committed:
Update docker-compose.yml

Add link to telescope config documentation/example and add `restart: unless-stopped` to the telescope container.

vcfxb commented:
> This needs a DNS rule set by @wdturner, and some more documentation on setting up the config (or a link to that in the Telescope repo)
> Aside from those, looks good!

I added a link, let me know if there is anything else :) 

EDIT: 2021/3/11 Wes added the DNS record we should be all set.
